README.md

```diff
@@ -82,20 +92,20 @@ var obj = {
   }
 }
 ```
-
--- YOUR ANSWER HERE --
+ES6
+Anonymous
+add(x, y)
 
 ## What does the following code output to the console?
 
 ```js
 var i = 0
-for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') ) { 
+for ( console.log('Bob'); (console.log('Curly') === undefined); console.log('Mo') ) {
   console.log(++i)
   if (i >= 2)
     break
 }
 console.log(i)
 ```
-
--- YOUR ANSWER HERE --
-
+Bob
+false
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.